project_multifamily_beta/housing_characteristics/Census Region.tsv

```diff
@@ -0,0 +1,11 @@
+Dependency=Census Division	Option=Midwest	Option=Northeast	Option=South	Option=West	sample_count	sample_weight
```

Comment:
Is Census Region actually used for anything?

Reply:
Dep graph shows no

Reply:
You are correct. Census Region is not being used for anything but is included for aggregation purposes.

Reply:
Seems like an uncommon aggregation to me. I'd advocate removing it just for results.csv file size reasons, but ok if you want to leave it.